apps/full-site-editing/full-site-editing-plugin/editor-gutenboarding-launch/index.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * File for various functionality which needs to be added to Simple and Atomic
+ * sites. The code in this file is always loaded in the block editor.
+ *
+ * Currently, this module may not be the best place if you need to load
+ * front-end assets, but you could always add a separate action for that.
+ *
+ * @package A8C\FSE
+ */
+
+namespace A8C\FSE\EditorGutenboardingLaunchButton;
+
+/**
+ * Enqueue assets
+ */
+function enqueue_script_and_style() {
+	// @TODO: Remove this block to enable in production
+	// Constant to disable the feature for development.
+	if ( ! ( defined( 'A8C_FSE_DOMAIN_PICKER_ENABLE' ) && A8C_FSE_DOMAIN_PICKER_ENABLE ) ) {
+		return;
+	}
+
+	// Avoid loading assets if possible.
+	if ( ! \A8C\FSE\Common\is_block_editor_screen() ) {
+		return;
+	}
+
+	$asset_file          = include plugin_dir_path( __FILE__ ) . 'dist/editor-gutenboarding-launch.asset.php';
+	$script_dependencies = isset( $asset_file['dependences'] ) ? $asset_file['dependencies'] : array();
+	$script_version      = isset( $asset_file['version'] ) ? $asset_file['version'] : filemtime( plugin_dir_path( __FILE__ ) . 'dist/editor-gutenboarding-launch.js' );
+	$style_version       = isset( $asset_file['version'] ) ? $asset_file['version'] : filemtime( plugin_dir_path( __FILE__ ) . 'dist/editor-gutenboarding-launch.css' );
+
+	wp_enqueue_script(
+		'a8c-fse-editor-gutenboarding-launch-script',
+		plugins_url( 'dist/editor-gutenboarding-launch.js', __FILE__ ),
+		$script_dependencies,
+		$script_version,
+		true
+	);
+
+	wp_enqueue_style(
+		'a8c-fse-editor-gutenboarding-launch-style',
+		plugins_url( 'dist/editor-gutenboarding-launch.css', __FILE__ ),
+		array(),
+		$style_version,
```

Comment:
This trailing comma is causing a fatal error for me locally:
`Parse error: syntax error, unexpected ')' in wp-calypso/apps/full-site-editing/full-site-editing-plugin/editor-gutenboarding-launch/index.php on line 47`

Reply:
What's your PHP version locally?

This should work with PHP 7.3 I think.

Reply:
Thanks for spotting that, we need to support PHP 7.2:

Remove the trailing comma: https://github.com/Automattic/wp-calypso/pull/42918